src/pages/mentees/[menteeId].tsx

```diff
@@ -33,16 +35,18 @@ export default widePage(() => {
       menteeId: userId,
       includeEndedTransactional: false,
     });
+  const { data:mapData } = trpcNext.map.listLandmarks.useQuery([...Latitudes]);
 
   return !u ? <Loader /> : <>
     <PageBreadcrumb current={`${formatUserName(u.name)}`} />
-    <MenteeTabs mentee={u} mentorships={mentorships || []} />
+    <MenteeTabs mentee={u} mentorships={mentorships || []} mapData={mapData || {}} />
   </>;
 });
 
-function MenteeTabs({ mentee, mentorships }: {
+function MenteeTabs({ mentee, mentorships, mapData }: {
```

Comment:
because the map feature isn't production ready. please hide the map tab and avoid fetching map data in production mode.